{"name": "test_redirect_to_personal_area_authorized_user", "status": "failed", "statusDetails": {"message": "AssertionError: Пользователь не перешел в личный кабинет\nassert False\n +  where False = <bound method PersonalAccount.is_at_personal_area of <page.personal_account_page.PersonalAccount object at 0x1066042e0>>()\n +    where <bound method PersonalAccount.is_at_personal_area of <page.personal_account_page.PersonalAccount object at 0x1066042e0>> = <page.personal_account_page.PersonalAccount object at 0x1066042e0>.is_at_personal_area", "trace": "self = <test_personal_account.TestPersonalAccount object at 0x1063f13d0>\nopen_browser = <selenium.webdriver.chrome.webdriver.WebDriver (session=\"24b069560076bc4b8d1d0580f635d298\")>\n\n    def test_redirect_to_personal_area_authorized_user(self, open_browser):\n        header = HeaderPage(open_browser)\n        personal_account = PersonalAccount(open_browser)\n        header.go_to_personal_account()\n>       assert personal_account.is_at_personal_area(), \"Пользователь не перешел в личный кабинет\"\nE       AssertionError: Пользователь не перешел в личный кабинет\nE       assert False\nE        +  where False = <bound method PersonalAccount.is_at_personal_area of <page.personal_account_page.PersonalAccount object at 0x1066042e0>>()\nE        +    where <bound method PersonalAccount.is_at_personal_area of <page.personal_account_page.PersonalAccount object at 0x1066042e0>> = <page.personal_account_page.PersonalAccount object at 0x1066042e0>.is_at_personal_area\n\ntests/test_personal_account.py:17: AssertionError"}, "steps": [{"name": "Клик по элементу: ('xpath', \"//div[contains(@class,'AppHeader_header__logo')]/a\")", "status": "passed", "steps": [{"name": "Ожидание видимости элемента: ('xpath', \"//div[contains(@class,'AppHeader_header__logo')]/a\")", "status": "passed", "parameters": [{"name": "locator", "value": "('xpath', \"//div[contains(@class,'AppHeader_header__logo')]/a\")"}], "start": 1729709345527, "stop": 1729709345536}], "parameters": [{"name": "locator", "value": "('xpath', \"//div[contains(@class,'AppHeader_header__logo')]/a\")"}, {"name": "expected_element", "value": "None"}], "start": 1729709345527, "stop": 1729709345732}, {"name": "Проверка, что элемент виден: None", "status": "passed", "steps": [{"name": "Ожидание видимости элемента: None", "status": "broken", "statusDetails": {"message": "TypeError: wait_for_element() takes 2 positional arguments but 3 were given\n", "trace": "  File \"/Users/artemkrivoshein/Diplom/Diplom_3/venv/lib/python3.9/site-packages/allure_commons/_allure.py\", line 192, in impl\n    return func(*a, **kw)\n"}, "parameters": [{"name": "locator", "value": "None"}], "start": 1729709345732, "stop": 1729709345732}], "parameters": [{"name": "locator", "value": "None"}, {"name": "timeout", "value": "10"}], "start": 1729709345732, "stop": 1729709345732}], "start": 1729709345527, "stop": 1729709345733, "uuid": "6e5a9ca4-4b54-40ed-b923-5e06e431d454", "historyId": "aaa8b77bd547bb4dbb6ea31a2538c5ab", "testCaseId": "aaa8b77bd547bb4dbb6ea31a2538c5ab", "fullName": "tests.test_personal_account.TestPersonalAccount#test_redirect_to_personal_area_authorized_user", "labels": [{"name": "tag", "value": "@pytest.mark.usefixtures('login')"}, {"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_personal_account"}, {"name": "subSuite", "value": "TestPersonalAccount"}, {"name": "host", "value": "MacBook-Air-Artem.local"}, {"name": "thread", "value": "64230-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_personal_account"}]}